wallet/src/model.rs

```diff
@@ -48,6 +48,13 @@ pub struct Balance {
     pub total: String,
 }
 
+#[derive(Debug, Serialize)]
+pub struct ExtendedKeyedSignature {
+    pub signature: String,
+    pub public_key: String,
+    pub chaincode: String,
```

Comment:
When `extended_pk` is set to false, this field is serialized as `"chaincode":""`. It is possible to not serialize it by using a serde attribute:

```suggestion
    #[serde(skip_serializing_if = "is_default")]
    pub chaincode: String,
```

But you need to define the `is_default` function:

```rust
fn is_default<T: Default + PartialEq>(t: &T) -> bool {
    t == &T::default()
}
```